src/test/java/org/variantsync/vevos/simulation/VariantGenerationTest.java

```diff
@@ -111,6 +111,7 @@ public boolean generate(final List<Variant> variantsToTest, final boolean writeC
     @BeforeClass
     public static void setupStatic() {
         VEVOS.Initialize();
+        Result.HARD_CRASH_ON_TRY = true;
```

Comment:
I get the following error on this line: `java: cannot assign a value to final variable HARD_CRASH_ON_TRY`
I executed the ./build-local-repo.sh before, so I should have the up-to-date version of the _functjonal_ library.

Reply:
Hmmm, but on the main branch of Functjonal, this field is not final: https://github.com/VariantSync/Functjonal/blob/main/src/main/java/org/variantsync/functjonal/Result.java

Are there any caching issues in maven?

Reply:
The jar has to be updated. I will do that.

Reply:
I updated the local maven repo and jar file now. Could you check if the error is gone now for you?

Reply:
Yes, it is working now.